### ISSUE TITLE
Compatibility with Colab?

### ISSUE THREAD

**Issue by kmedved** [evhub/bbopt#15]
Is Bbopt compatible with Jupyter Notebooks? I'm trying to test it in Colab, but I'm having issues getting it to work outside of a .py format.

Thanks for the help and the very cool package.

**Comment by evhub** [comment]
@kmedved It works in Jupyter notebooks—the problem is that Colab specifically doesn't let you write to local files. It looks like if you want to save state in a Colab notebook, you're supposed to write to Google Drive. I'll leave this issue open for adding an option to write to Google Drive.

**Comment by kmedved** [comment]
Thanks. Would it be possible to put add an example of notebook functionality? I'm getting tripped up on the `bb = BlackBoxOptimizer(file=__file__)` line in the examples, which seems more geared for a traditional .py script with a `if __name__ == "__main__":` interface.

Apologies for the somewhat simplistic question. Very eager to try this package.

**Comment by evhub** [comment]
Yeah, I'll add some docs on that. For now, you should be able to just do:
```python
import os
from bbopt import BlackBoxOptimizer

bb = BlackBoxOptimizer(file=os.path.join(os.getcwd(), "my_project_name"))
```

**Comment by kmedved** [comment]
Thank you!
